### ISSUE TITLE
The two json files shouldn't be linked to each other directly but through the model/system they came from.

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy#190]
The abstraction here is that we have a System (or model). That same model can have shapes attached to it in variety of ways, e.g. you can have one pendulum model but may want to model the bobs as stars in one scene or spheres in another. Or maybe you want a blue pendulum scene and red pendulum scene but they both use the same underlying model. Then you have simulations, i.e. the trajectory of the system's states. These are independent of the scenes but tied to the model, so you should be able to use the same simulation data for both the red and blue pendulum scenes because they all derive from the same model. So the two files need to have a unique identifier that ties it to a Scene object in Python. It isn't clear that this is going on, because a new scene is created every time scene.display() is called. It should look for the scene and if it matches the python side, not regenerate it. Only when the scene.json file changes from the one prescribed in python should it be potentially saved as a new scene file.

**Comment by tarzzz** [comment]
A possible solution for this issue would be to use a unique indentifier. Currently, by default we are using timestamp as the identifier to correlate the simulation file and scene file. We can use an id, specific to simulation data, which can be used for multiple scene files (one with star bob, and another with spherical for instance). That id can be saved in the System class or somewhere, and can be re-used.

**Comment by moorepants** [comment]
Yes a unique identifier is needed. But I think there is a bigger picture issue here. We have a model, a scene that displays the model (potentially different shapes for the model), and the simulation data. A model can have many simulations and many sets of shapes. So an identifier needs to be associated with a model. Models are somewhat uniquely identified by there equations of motion. We could use that as the identifier. But identical models can have different but equivalent mathematical descriptions. One way to fine if a model is equivalent is to evaluate the rhs of the odes (given that the states, constants, and specifieds are defined the same, then the rhs should equate to close to machine precision. Plus evaluating the rhs is in general computationally cheap.
